From fa2ceb127060000d26758db36e5e13c4c7d97f77 Mon Sep 17 00:00:00 2001
From: sunshihao520 <sunshihao@huawei.com>
Date: Thu, 4 Feb 2021 19:49:03 +0800
Subject: [PATCH 03/15] pkg:spdk.spec Add ncurses-devel to BuildRequires

ncurses.h is needed in spdk_top, add ncurses-devel to BuildRequires.

Signed-off-by: sunshihao <sunshihao@huawei.com>
Reviewed-on: https://review.spdk.io/gerrit/c/spdk/spdk/+/6258 (master)

(cherry picked from commit ccbdd1750f3da9be85b2018896fab5c88cb7f686)
Change-Id: I22754838788fe237a29ab74a675a94ab7007bd33
Signed-off-by: Tomasz Zawadzki <tomasz.zawadzki@intel.com>
Reviewed-on: https://review.spdk.io/gerrit/c/spdk/spdk/+/6388
Tested-by: SPDK CI Jenkins <sys_sgci@intel.com>
Reviewed-by: Jim Harris <james.r.harris@intel.com>
Reviewed-by: sunshihao <sunshihao@huawei.com>
Reviewed-by: Aleksey Marchuk <alexeymar@mellanox.com>
---
 pkg/spdk.spec | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/pkg/spdk.spec b/pkg/spdk.spec
index 2237635..23de4f8 100644
--- a/pkg/spdk.spec
+++ b/pkg/spdk.spec
@@ -22,7 +22,7 @@ License: BSD
 ExclusiveArch: x86_64
 
 BuildRequires: gcc gcc-c++ make
-BuildRequires: dpdk-devel, numactl-devel
+BuildRequires: dpdk-devel, numactl-devel, ncurses-devel
 BuildRequires: libiscsi-devel, libaio-devel, openssl-devel, libuuid-devel
 BuildRequires: libibverbs-devel, librdmacm-devel
 %if %{with doc}
-- 
1.8.3.1

